File: Lista-15/java/3.java
public class ContaBancaria {
    private String titular;
    private double saldo;

    public ContaBancaria(String titular, double saldo) {
        this.titular = titular;
        this.saldo = saldo;
    }

    public void depositar(double valor) {
        this.saldo += valor;
    }

    public void sacar(double valor) {
        if (valor <= this.saldo) {
            this.saldo -= valor;
        } else {
            System.out.println("Saldo insuficiente");
        }
    }

    public String exibirSaldo() {
        return "Saldo atual: R$" + this.saldo;
    }

    public static void main(String[] args) {
        ContaBancaria conta = new ContaBancaria("João", 500);
        conta.depositar(100);
        System.out.println(conta.exibirSaldo());
        conta.sacar(200);
        System.out.println(conta.exibirSaldo());
    }
}

File: Lista-15/go/3.go
package main
import "fmt"

type ContaBancaria struct {
    Titular string
    Saldo   float64
}

func (c *ContaBancaria) Depositar(valor float64) {
    c.Saldo += valor
}

func (c *ContaBancaria) Sacar(valor float64) {
    if valor <= c.Saldo {
        c.Saldo -= valor
    } else {
        fmt.Println("Saldo insuficiente")
    }
}

func (c ContaBancaria) ExibirSaldo() string {
    return fmt.Sprintf("Saldo atual: R$%.2f", c.Saldo)
}

func main() {
    conta := ContaBancaria{Titular: "João", Saldo: 500}
    conta.Depositar(100)
    fmt.Println(conta.ExibirSaldo())
    conta.Sacar(200)
    fmt.Println(conta.ExibirSaldo())
}
